### ISSUE TITLE
bug: @ionic/react IonButton href navigation causes full window reload

### ISSUE THREAD

**Issue by gerwinkoudijs** [ionic-team/ionic#18888]
# Bug Report

**Ionic version:**

[x] **@ionic/react 0.0.7**

**Current behavior:**
A full page reload occurs when navigating to an internal link using the `href` property of the `IonButton` component. The screen goes white and shows the splash screen for a short amount of time. The navigation does work and ends up showing the right page.

This problem does not occurs when navigating using the IonTabButton.

My current workaround is to navigate programmatically (I use the `use-react-router` npm package for this).

**Expected behavior:**
A smooth navigation transition to the next page.

**Steps to reproduce:**
Just add an `IonButton` component and set the `href` attribute. The path should be added as a `Route` within the `IonRouterOutlet`.

**Ionic info:**

```
Ionic:

   Ionic CLI       : 5.2.3 (/Users/gerwinkoudijs/.nvm/versions/node/v10.15.1/lib/node_modules/ionic)
   Ionic Framework : @ionic/react 0.0.7

Capacitor:

   Capacitor CLI   : 1.1.1
   @capacitor/core : 1.1.1

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v10.15.1 (/Users/gerwinkoudijs/.nvm/versions/node/v10.15.1/bin/node)
   npm    : 6.4.1
   OS     : macOS Mojave

```

**Comment by elylucas** [comment]
Hi @gerwinkoudijs, this should be fixed in 0.0.9. Let us know if you still see the issue.

Thanks

**Comment by gerwinkoudijs** [comment]
Thanks. It works fine now!
